lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -766,11 +777,14 @@ def _post(self, params: ApiParams, _) -> Response:
                     klasname,
                     type(e).__name__,
                 )
-                raise APIAbort(HTTPStatus.INTERNAL_SERVER_ERROR)
+                raise APIAbort(
+                    HTTPStatus.INTERNAL_SERVER_ERROR,
+                    f"Unexpected backend error {str(e)!r}",
+                )
         elif self.require_map:
             raise APIAbort(
                 HTTPStatus.CONFLICT,
-                f"Dataset {self.action} requires Indexed state ({dataset.state.name})",
+                f"Dataset {self.action} requires Indexed dataset but state is {dataset.state.friendly}",
```

Comment:
Should `dataset.state.friendly` come out in quotation marks?

Reply:
I suppose I *could* quote both of them, but I didn't see the need.